features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2

```diff
@@ -33,5 +33,8 @@ recaptcha.site.key={{identity.recaptcha.site_key}}
 # reCaptcha secret key
 recaptcha.secret.key={{identity.recaptcha.secret_key}}
```

Comment:
Can we remove the "identity" prefix from all the keys relate to recaptcha. 
Eg: identity.recaptcha.secret_key to recaptcha.secret_key

Reply:
fixed